kube_apiserver_metrics/assets/dashboards/overview.json

```
@@ -428,7 +428,7 @@
                   "response_format": "scalar",
                   "queries": [
                     {
-                      "query": "sum:kube_apiserver.etcd_object_counts{$kube_cluster_name} by {resource}",
+                      "query": "sum:kube_apiserver.apiserver_storage_objects{$kube_cluster_name} by {resource}",
```

As above, I’m wondering if it shouldn’t be:
```suggestion
                      "query": "sum:kube_apiserver.storage_objects{$kube_cluster_name} by {resource}",
```